content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -162,6 +162,42 @@ are built in and may not be overwritten:
   that only matches the `catch-all` FlowSchema will be rejected with an HTTP 429
   error.
 
+## Potentially interesting additional configuration
+
+The suggested configuration gives no special treatment to the health
+check requests on kube-apiservers from their local kubelets --- which
```

Comment:
what is the downside of giving special treatment to health check requests?

Reply:
@MikeSpreitzer (from our slack chat): `Now, the down side is that this is ripping a hole in the intended function, which is protecting against abusive clients`

Reply:
i think it is ok to remove the protection from local kubelet. so the key here is there is no easy way currently to distinguish requests from local from non-local, right?

Reply:
right

Reply:
Hi @MikeSpreitzer . I read through the changes. A couple of questions:
- Does this section belong under another section such as `Exempt requests` or `FlowSchema`?
- I'd like to see one sentence to introduce this section, something like. 
To prevent rate limiting of health check requests between a kube-apiserver and their local kubelets,
you can add a FlowSchema that sets ...
- What is the `suggested configuration`?